Bottom: 15aa268a411d1cf7647765b20d21e0f2645d1ccf
Top:    3fae07ed3c6fdf10454a4a6928df490b3095b605
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-27 02:40:17 +0100

Refresh of avoid-hard-coding-the-number

---

diff --git a/src/const.h b/src/const.h
index 08fe253..67eb72d 100644
--- a/src/const.h
+++ b/src/const.h
@@ -42,6 +42,8 @@
 #define TOP_SIDE                1
 
 
+#define EXTRA_LAYERS 2 /* 2x silkscreen layers */
+
 /* ---------------------------------------------------------------------------
  * the layer-numbers of the two additional special (silkscreen) layers
  * 'bottom' and 'top'. The offset of MAX_LAYER is not added
diff --git a/src/data.h b/src/data.h
index 285ab2c..5d40d71 100644
--- a/src/data.h
+++ b/src/data.h
@@ -45,7 +45,6 @@ extern OutputType Output;
 
 extern PCBType *PCB;
 
-#define EXTRA_LAYERS 2 /* 2x silkscreen layers */
 #define max_group (PCB->Data->LayerN)
 #define max_copper_layer (PCB->Data->LayerN)
 #define bottom_silk_layer (max_copper_layer + BOTTOM_SILK_LAYER)
